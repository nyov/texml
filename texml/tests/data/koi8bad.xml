<?xml version="1.0" encoding="koi8-r"?>
<TeXML>Плохие символы: &#8764;&#8765;</TeXML>
