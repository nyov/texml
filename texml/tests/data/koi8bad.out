Плохие символы: &#x77D;&#x7D4;